### ISSUE TITLE
Doesn't work in Windows 10

### ISSUE THREAD

**Issue by AerialAtom** [gen2brain/cbconvert#1]
Can't add files either from dragging (red no symbols) or by adding files or directories. 
![screenshot 73](https://user-images.githubusercontent.com/10418462/27313119-12167fbc-5521-11e7-9652-80dca85e0143.png)

**Comment by gen2brain** [comment]
I will probably have to rebuild app with newer Qt. There can be problems with go-qml library, it looks abandoned.
What happens when you try Add Files/Add Dir buttons?

**Comment by AerialAtom** [comment]
Nothing happens. It doesn't add files or directories. I have tried it on two Windows computer, same results.

**Comment by seacatwu** [comment]
same problem in the  windows 10

**Comment by gen2brain** [comment]
I will have to rewrite app with different library, the library it was used before is broken with new versions of Go language. It is a simple UI but still will need some time for that.

**Comment by 03stevensmi** [comment]
> I will have to rewrite app with different library, the library it was used before is broken with new versions of Go language. It is a simple UI but still will need some time for that.

Is this STILL Happening? It's Been Like 3 and a half YEARS. Any Updates?

**Comment by gen2brain** [comment]
No, still no update, I just use command line app to convert comic books.

**Comment by rohitghali** [comment]
There's a UI? How do I get the UI?

**Comment by gen2brain** [comment]
@rohitghali There was a UI, it was removed from the repo. If I find time I will rewrite it, that is why this issue is still open.

**Comment by rohitghali** [comment]
> @rohitghali There was a UI, it was removed from the repo. If I find time I will rewrite it, that is why this issue is still open.

Why was it removed? I mean, it looks pretty good in the image above. Any issues?

**Comment by gen2brain** [comment]
I am closing this, if I am ever to rewrite a GUI I can drop a comment here.

**Comment by gen2brain** [comment]
I wrote the new GUI, release has been uploaded. Now it uses the OS native controls so it should last longer.
